pkgs/applications/audio/pifi/default.nix

```
@@ -0,0 +1,18 @@
+{ lib, bundlerEnv, ruby }:
+
+bundlerEnv rec {
+  name = "pifi-${version}";
+
+  version = (import ./gemset.nix).pifi.version;
+  inherit ruby;
+  # expects Gemfile, Gemfile.lock and gemset.nix in the same directory
+  gemdir = ./.;
+
+  meta = with lib; {
+    description = "MPD web client to listen to radio, written in React and Sinatra.";
+    homepage = "https://github.com/rccavalcanti/pifi-radio";
+    license = with licenses; gpl3;
```

The license is [GPLv3](https://github.com/rccavalcanti/pifi-radio/blob/master/pifi.gemspec#L7) only.

```suggestion
    license = with licenses; gpl3Only;
```